lms/djangoapps/teams/static/teams/templates/edit-team.underscore

```diff
@@ -0,0 +1,21 @@
+<div class="wrapper-msg is-incontext urgency-low warning">
+    <div class="msg">
+        <div class="msg-content">
+            <h3 class="title"><%= gettext("Oops!") %></h3>
```

Comment:
Thanks @frrrances I'll huddle with the Doc team to come up with a plan so that we all make consistent recommendations.